R/statistic-histogram.R

```diff
@@ -50,13 +50,13 @@
 #' @return Accuracy guarantee for histogram release, given epsilon.
 #' @rdname histogram.getAccuracy
 
-histogram.getAccuracy <- function(mechanism, n.bins, n, epsilon, delta=10^-6, alpha=0.05) {
+histogram.getAccuracy <- function(mechanism, n.bins, n, epsilon, delta=10^-6, alpha=0.05, sensitivity) {
  	acc <- NULL
 	if(mechanism == 'mechanismStability'){
 		acc <- (2*log(2/(alpha*delta)) /epsilon) + 1
```

Comment:
Minor, but would it make sense to extract out a `stability.getAccuracy`, and `stability.getEpsilon` as well, for homogeneity?

Reply:
I like that idea